.gitignore

```diff
@@ -6,5 +6,6 @@ packages/testing/*/lib
 .idea
 coverage/
 tsconfig.build.tsbuildinfo
+.yarn
```

Comment:
Is there a motive for `.yarn` to be included as part of `.gitignore?`

Reply:
Yes, yarn 3 created `.yarn` directory

Reply:
@ilyalatt You should use yarn 2 with this repo :/ we do not intend to use yarn 3, in fact the intention is to switch to PNPM

Reply:
Ok, I have reverted this change